src/main/java/org/folio/rest/impl/AcquisitionsUnitsApi.java

```diff
@@ -63,8 +63,8 @@ public void getAcquisitionsUnitsUnits(String query, String totalRecords, int off
 
     acquisitionsUnitsService.getAcquisitionsUnits(query, offset, limit, new RequestContext(vertxContext, okapiHeaders))
       .onSuccess(units -> {
-        if (logger.isInfoEnabled()) {
-          logger.info("Successfully retrieved acquisitions units: {}", JsonObject.mapFrom(units).encodePrettily());
+        if (logger.isDebugEnabled()) {
+          logger.debug("Successfully retrieved acquisitions units: {}", JsonObject.mapFrom(units).encodePrettily());
```

Comment:
remained only CREATE, UPDATE, DELETE operations with into level to not have noise in logs